images/crane/tests/01-version.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset -o errtrace -o pipefail -x
+
+if [[ "${IMAGE_NAME}" == "" ]]; then
+    echo "Must set IMAGE_NAME environment variable. Exiting."
+    exit 1
+fi
+
+docker run --rm "${IMAGE_NAME}" version
```

Comment:
Would be good to add a bunch of tests since they are relatively cheap with `crane` (e.g. `manifest`, `digest`, etc.)